_commands/caen/Caen-Connect-Tutorial.md

```diff
@@ -0,0 +1,149 @@
+---
+---
+
+Caen-Connect-Tutorial
```

Comment:
Small note, but I'd prefer this without the hyphens `-` unless you have a specific reason for including them.